PowerVCF.psm1

```diff
@@ -939,13 +929,7 @@ Function Request-VCFCertificate {
         [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$json,
         [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$domainName
     )
-
-    if (!(Test-Path $json)) {
-        Throw "JSON File Not Found"
-    }
-    else {
-        # Reads the requestCsrSpec json file contents into the $ConfigJson variable
-        $ConfigJson = (Get-Content -Raw $json)
+        validateJsonInput
```

Comment:
Slipped in?

Reply:
d'oh! damn it, yes, slipped yet again from the other branch. sorry, let me restore it to how it was

Reply:
fixed